doc/progress.rst

```diff
@@ -21,6 +21,7 @@ Changelog
 * ADD #659: Lazy loading of task splits.
 * ADD #516: `run_flow_on_task` flow uploading is now optional.
 * ADD #680: Adds `openml.config.start_using_configuration_for_example` (and resp. stop) to easily connect to the test server.
+* ADD #695: A function to retieve all the data quality measures available.
```

Comment:
Please add new additions under a new header (just call the header 'newest release' for now). The 0.9.0 release is already out, so this change is not contained in it.